app/assets/stylesheets/_tabs.scss

```diff
@@ -0,0 +1,112 @@
+.entries-tabs-container{
+  margin-left: -25px;
+  margin-right: -25px;
+}
+
+.reports-tabs-container {
+  @include media(max-width $medium-screen) {
+    width:100%;
+  }
+  width:73%;
+}
+
+.accordion-tabs {
+  @include clearfix;
+  line-height: 1.5;
+  margin-bottom: $base-spacing;
+
+  @include media(max-width $medium-screen) {
+    border: $tab-border;
+    border-radius: $base-border-radius;
+  }
+
+  .tab-header-and-content {
+    @include media($medium-screen) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets